README.md

```diff
@@ -79,7 +79,33 @@ npm run dev
 http://localhost:3000/
 ```
 
-## 🎉 How to deploy on Azure for [free](<(https://azure.microsoft.com/products/container-apps#Pricing)>)
+## 🎉 How to deploy on Azure with [free grants](<(https://azure.microsoft.com/products/container-apps#Pricing)>)
+
+Jhipster Azure Container Apps provides multiple ways to deploy on Azure, you can deploy with the built-in [Terraform](https://www.terraform.io/), [Bicept](https://learn.microsoft.com/azure/azure-resource-manager/bicep/overview?tabs=bicep), [Azure Developer CLI(AZD)](https://learn.microsoft.com/azure/developer/azure-developer-cli/) or directly from [source code](https://learn.microsoft.com/azure/container-apps/java-get-started?pivots=jar).
+
+### Terraform
+
+1. Make sure you marked `Terraform` when generating the project.
+1. Make sure you have [Terraform](https://learn.microsoft.com/azure/developer/terraform/quickstart-configure) installed and configured.
+1. Change into the Terraform folder: `cd terraform`
+1. Run the following command to initialize Terraform:
+   </br> `terraform init`
+1. Once finished, privision the necessary resource on Azure with:
+   </br> `terraform apply -auto-approve`
+1. Now you can deploy the project with:
```

Comment:
deploy.ps1 is only for Windows.

We should mention deploy.sh, which is for Linux/MacOS.

usage:  /deploy.sh subId region resourceGroupName